aria_data_loaders/aria_data_utils/aria_streamer.py

```diff
@@ -1,25 +1,21 @@
 import logging
 import os
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Tuple
 
 import click
 import cv2
 import numpy as np
 import sophus as sp
 from aria_data_utils.detector_wrappers.april_tag_detector import AprilTagDetectorWrapper
 from aria_data_utils.detector_wrappers.object_detector import ObjectDetectorWrapper
-from aria_data_utils.image_utils import decorate_img_with_text
-from aria_data_utils.perception.april_tag_pose_estimator import AprilTagPoseEstimator
-from bosdyn.client.frame_helpers import get_a_tform_b
 from fairotag.scene import Scene
 from matplotlib import pyplot as plt
 from projectaria_tools.core import calibration, data_provider, mps
 from scipy.spatial.transform import Rotation as R
 from spot_rl.envs.skill_manager import SpotSkillManager
-from spot_rl.models.owlvit import OwlVit
-from spot_wrapper.spot import Spot, SpotCamIds, image_response_to_cv2
+from spot_wrapper.spot import Spot
+from spot_wrapper.spot_qr_detector import SpotQRDetector
```

Comment:
@jimmytyyang  is it okay to move SpotQRDetector code inside bd_spot_wrapper?

Reply:
yeah, I think it should be fine